homeassistant/components/fyta/sensor.py

```diff
@@ -138,7 +145,7 @@ async def async_setup_entry(
         FytaPlantSensor(coordinator, entry, sensor, plant_id)
         for plant_id in coordinator.fyta.plant_list
         for sensor in SENSORS
-        if sensor.key in coordinator.data[plant_id]
+        if sensor.key in dir(coordinator.data[plant_id])
```

Comment:
What does this check? Should we check if value_fn(plant) is not None? (this probably also works, be sure to have a test for this)

Reply:
It makes sure that sensors are only set up in case we have the corresponding attribute in Plant. If this is always the case, it could be omitted.
Regarding none-values, I added this in the test for ph (which may or may not be set by the user and therefore can be none).

Reply:
But then isn't that contradicting?

Reply:
I don't think so. It's not testing, if the attribute has a specific value, but only if it exists.

Reply:
Would you suggest to delete the line (i.e. are you confident that there always are all attributes available)?